src/main/scala/sbtghpackages/TokenSource.scala

```diff
@@ -23,6 +23,7 @@ sealed trait TokenSource extends Product with Serializable {
 
 object TokenSource {
   final case class Environment(variable: String) extends TokenSource
+  final case class Property(key: String) extends TokenSource
   final case class GitConfig(key: String) extends TokenSource
   final case class Or(primary: TokenSource, secondary: TokenSource) extends TokenSource
```

Comment:
Nice to Have. A `Or` method with the three parameters.

Reply:
Two parameters should be enough:
```scala
val source: TokenSource = TokenSource.Environment("TOKEN_1") || TokenSource.GitConfig("config") ||  
TokenSource.Property("PROPERTY_TOKEN") || TokenSource.Environment("TOKEN_2")
```

Encoded as:
```scala
val source: TokenSource = TokenSource.Or(
  TokenSource.Environment("TOKEN_1"),
  TokenSource.Or(
    TokenSource.GitConfig("config"),
    TokenSource.Or(
      TokenSource.Property("PROPERTY_TOKEN"),
      TokenSource.Environment("TOKEN_2")
    )
  )
)
```

Reply:
Yes, good point. That would work.